src/pages/loading-waiting-states/spinner-overlays/index.tsx

```diff
@@ -0,0 +1,197 @@
+        left: 0,
+        width: '100%',
+        height: '100%',
+        background: Colors.white[200],
+        zIndex: 3,
+        opacity: 0.2,
+    },
+    btnContainerStyle: {
+        padding: `${theme.spacing(3)}px`,
+    },
+    reloadButtonStyle: {
+        width: '100%',
+    },
+    eulaContentStyle: {
+        paddingTop: `${theme.spacing(1)}px`,
+    },
+    eulaConfirmationCheckStyle: {
+        padding: `0 ${theme.spacing(3)}px`,
+    },
+}));
+
+export const SpinnerOverlays = (): JSX.Element => {
+    const dispatch = useDispatch();
+    const theme = useTheme();
+    const classes = useStyles();
+    const spinnerCirlceSize = 96;
+    const [checkboxState, setCheckboxState] = useState(false);
+    const [eulaLoaded, setEulaLoaded] = useState(false);
+
+    const changeCheckboxState = (event: React.ChangeEvent<HTMLInputElement>): void => {
+        setCheckboxState(event.target.checked);
+    };
+
+    useEffect(() => {
+        let isEulaLoaded = true;
+
+        setTimeout(() => {
+            if (isEulaLoaded) {
+                setEulaLoaded(true);
+            }
+        }, 3000);
+
+        return (): void => {
+            isEulaLoaded = false;
+        };
+    }, [checkboxState, eulaLoaded, setEulaLoaded]);
+
+    const reloadEulaDetails = useCallback((): void => {
+        setEulaLoaded(false);
+        setCheckboxState(false);
+    }, [checkboxState, eulaLoaded, setEulaLoaded]);
+
+    return (
+        <div style={{ backgroundColor: theme.palette.background.paper, minHeight: '100vh' }}>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp={true}>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        Spinner Overlays
+                    </Typography>
+                    <div />
+                </Toolbar>
+            </AppBar>
+            <div className={classes.container}>
+                <Card className={classes.eulaContainerStyle}>
+                    {!eulaLoaded && (
+                        <div className={classes.loadingOverlayContainerStyle}>
+                            <CircularProgress color="secondary" size={spinnerCirlceSize} />
+                        </div>
+                    )}
+                    <div className={`${classes.eulaDetailsStyle} ${!eulaLoaded ? classes.overlayContainerStyle : ''}`}>
```

Comment:
Look into the CardHeader, CardContent, and CardAction components from Material UI for organizing the content of the card. The Eula part should scroll, but the Header should remain in place. You can also reference the implementation of this screen from the Auth Workflow (it's a little complex because different pieces are in different files, but the pagers handle most of the layout): https://github.com/pxblue/react-workflows/blob/dev/login-workflow/src/screens/InviteRegistrationPager.tsx#L418